### PR TITLE
Update azuredeploy.json

### PR DESCRIPTION
**Pull request by alexlzx**
# PR Checklist

Check these items before submitting a PR... 

[Contribution Guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/README.md)

[Best Practice Guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/best-practices.md)

- [ ] - Please check this box once you've submitted the PR if you've read through the Contribution Guide and best practices checklist.

## Changelog

*
*
*

### CODE REVIEW

**Comment by azure-quickstarts** [comment]
@mamccrea - check this PR for updates that may be needed to documentation that references this sample.  [This is an automated message. You are receiving it because you are listed as the docOwner in metadata.json.]

**Review by ruiminwang — APPROVED** [review]

File: quickstarts/microsoft.streamanalytics/streamanalytics-create/azuredeploy.json [hunk 1]
```diff
@@ -27,21 +27,16 @@
...
-        18,
-        24,
-        30,
-        36,
-        42,
-        48
+        7,
+        10,
+        20,
+        30
```
Comment (ruiminwang) [inline quickstarts/microsoft.streamanalytics/streamanalytics-create/azuredeploy.json:34]: Actually the allowed values could be [3, 7, 10, 20, 30, 40 ,50 ..... up to 660]

File: quickstarts/microsoft.streamanalytics/streamanalytics-create/azuredeploy.json [hunk 2]
```diff
@@ -27,21 +27,16 @@
...
       ],
-      "maxValue": 48,
-      "minValue": 1,
+      "maxValue": 660,
+      "minValue": 3,
       "metadata": {
-        "description": "Number of Streaming Units"
+        "description": "You can choose the number of Streaming Units, ranging from 3, 7, 10, 20, 30, and so on, up to 660."
```
Comment (ruiminwang) [inline quickstarts/microsoft.streamanalytics/streamanalytics-create/azuredeploy.json:39]: 3, 7, 10, 20, ..., multiples of 10 up to 660.

**Comment by alexlzx** [comment]
@mamccrea Can you help us merge this pull request? Thanks

**Comment by alexlzx** [comment]
I don't see the update shown on the file. Can you please verify if my PR was merged successfully? [azuredeploy.json](https://github.com/Azure/azure-quickstart-templates/blob/master/quickstarts/microsoft.streamanalytics/streamanalytics-create/azuredeploy.json)

**Comment by anthony-c-martin** [comment]
@alexlzx looks like this sample is generated from a `.bicep` file: https://github.com/Azure/azure-quickstart-templates/blob/master/quickstarts/microsoft.streamanalytics/streamanalytics-create/main.bicep

After your PR was merged, looks like our automation re-generated the azuredeploy.json file and overwrote your changes: https://github.com/Azure/azure-quickstart-templates/commit/3dc1b4fa9833b8571e5390b5272eb3b1e7c14b65

Sorry, I missed that when reviewing - I assumed automation in this repo would have blocked this scenario. I'll raise an issue to improve the automation.

To make the change successfully, you will need to edit the `main.bicep` file instead and submit a new PR for this. You don't need to include the changes to the azuredeploy.json file - they will be automatically made after your PR is merged.

**Comment by anboisve** [comment]
@anthony-c-martin 

We have updated the main.bicep file and the json file has been updated -- however, the doc we are trying to update is still showing previous info: https://learn.microsoft.com/en-us/azure/stream-analytics/quick-create-azure-resource-manager#review-the-template

Please advise on next steps.

CC @alexlzx 
CC @spelluru also if you can please help.

**Comment by spelluru** [comment]
@anthony-c-martin 

That template is not hard-coded. It's showing the template from the following one. Looking at the quickstart template PR, that's the file which was updated.

 :::code language="json" source="~/quickstart-templates/quickstarts/microsoft.streamanalytics/streamanalytics-create/azuredeploy.json":::

I have created a [PR for the azure-docs-pr repo](https://github.com/MicrosoftDocs/azure-docs-pr/pull/249652) and verified that the new template shows up on staging. The PR is merged and we should see the updated template in the article after 10 AM PST publish today

**Pull request merged**